python/lsst/scarlet/lite/operators.py

```diff
@@ -395,8 +395,8 @@ def uncentered_operator(
     if py == cy and px == cx:
         return func(x, **kwargs)
 
-    dy = int(2 * (py - cy))
-    dx = int(2 * (px - cx))
+    dy = int(round(2 * (py - cy)))
```

Comment:
Just making sure you want to round after doubling here, in case you do/don't want even dimensions.